server/worktree/status.go

```
@@ -0,0 +1,75 @@
+			kind = serverapi.WorktreeStatusProblemRootMissing
+		}
+		problemRoot := root
+		response.Problems = append(response.Problems, serverapi.WorktreeStatusProblem{Kind: kind, Root: &problemRoot})
+		return response, nil
+	}
+	observed, observedErr := s.git.InspectTarget(ctx, root)
+	workspace, workspaceErr := s.git.InspectTarget(ctx, target.WorkspaceRoot)
+	if observedErr != nil || workspaceErr != nil {
+		problemRoot := root
+		response.Problems = append(response.Problems, serverapi.WorktreeStatusProblem{Kind: serverapi.WorktreeStatusProblemGitBindingMissing, Root: &problemRoot})
+		return response, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 54) but it returns nil

(nilerr)